tests/e2e/specs/miscellaneous/WorkspaceWithParent.spec.ts

```diff
@@ -32,9 +32,14 @@ suite('Workspace using a parent test suite', function (): void {
 	const kubernetesCommandLineToolsExecutor: KubernetesCommandLineToolsExecutor = e2eContainer.get(
 		CLASSES.KubernetesCommandLineToolsExecutor
 	);
+
 	let podName: string = '';
+
+	suiteSetup(function (): void {
+		kubernetesCommandLineToolsExecutor.loginToOcp();
```

Comment:
It can be configured here: https://github.com/eclipse/che/blob/main/tests/e2e/constants/API_TEST_CONSTANTS.ts#L13
By default, it uses OC client: https://github.com/eclipse/che/blob/main/tests/e2e/constants/API_TEST_CONSTANTS.ts#L13

Reply:
Got it, thanks for the explanation, @musienko-maxim !